docker/ebpf-probe-builder/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM debian:unstable
+
+MAINTAINER Sysdig <support@sysdig.com>
+
+ENV SYSDIG_HOST_ROOT /host
+
+ENV HOME /root
+
+# Based on the sysdig container, used for building eBPF probe
+
+RUN apt-get update \
+ && apt-get dist-upgrade -y \
+ && apt-get install -y --no-install-recommends \
+	clang-7 \
+	gcc \
+	libelf-dev \
+	libelf1 \
+	llvm-7 \
+	make \
+ && rm -rf /var/lib/apt/lists/*
+
+# Use clang-7 as the default clang
+RUN rm -rf /usr/bin/clang \
+ && rm -rf /usr/bin/llc \
+ && ln -s /usr/bin/clang-7 /usr/bin/clang \
+ && ln -s /usr/bin/llc-7 /usr/bin/llc
+
+RUN rm -df /lib/modules \
```

Comment:
Same with this, it shouldn't be needed anymore inside this image, unless I'm missing something.